src/Faker/Provider/fr_FR/PhoneNumber.php

```diff
@@ -81,6 +88,24 @@ public function phoneNumber07WithSeparator()
         return $phoneNumber;
     }
 
+    public function phoneNumber08()
+    {
+        $phoneNumber = $this->phoneNumber08WithSeparator();
+        $phoneNumber = str_replace(' ', '', $phoneNumber);
+        return $phoneNumber;
+    }
+
+    /**
+     *
+     * @see https://www.arcep.fr/index.php?id=8146#c9625
+     * @see https://issuetracker.google.com/u/1/issues/73269839
+     */
+    public function phoneNumber08WithSeparator()
+    {
+        $regex = '([012]{1}\d{1}|(9[1-357-9])( \d{2}){3}';
```

Comment:
please document it better to it's easier to modify in the future if the rules change

Reply:
I added the formats as documentation, the missing 4|6 numbers aren't attributed yet.